keras/activations.py

```diff
@@ -61,30 +79,74 @@ def selu(x):
 
 
 def softplus(x):
+    """Softplus activation function.
+
+    # Arguments
+        x: Input tensor
+
+    # Returns
+        The softplus activation: `log(exp(x) + 1`.
+    """
     return K.softplus(x)
 
 
 def softsign(x):
+    """Softsign activation function.
+
+    # Arguments
+        x: Input tensor
+
+    # Returns
+        The softplus activation: `x / (abs(x) + 1)`
+    """
     return K.softsign(x)
 
 
 def relu(x, alpha=0., max_value=None):
+    """(Leaky) Rectified Linear Unit.
+
+    # Arguments
+        - x: Input tensor
```

Comment:
You normally end one-line descriptions with a period, so do it here too for consistency